src/components/StreamBanner/StreamBanner.js

```diff
@@ -47,7 +47,7 @@ export default function StreamBanner ({
     ({ bannerUrl, avatarUrl, name, location } = group)
   }
 
-  let numCustomFilters = isCustomView ? (customPostTypes.length + customViewTopics.length + (customActivePostsOnly ? 1 : 0)) : false
+  let numCustomFilters = customViewType === 'stream' ? (customPostTypes.length + customViewTopics.length + (customActivePostsOnly ? 1 : 0)) : false
```

Comment:
Realizing you're adding to this not creating it, I would like to see this broken down more and on more lines. Maybe a couple `is` / `has` sorta variables. Also this should be a `const`